SpywareFilter/sections/specific.txt

```diff
@@ -22,6 +22,9 @@
 ||nd.demdex.net/event?d_mid=$redirect=nooptext,important,domain=9now.com.au
 ||adc.nine.com.au/?token=$redirect=nooptext,important,domain=9now.com.au
 !
+||res.heraldm.com/js/acecounter_*.js
+||res.heraldm.com/js/logcollectscript_*.js
+||herald.loginside.co.kr/?
```

Comment:
I guess it should be `||herald.loginside.co.kr^` in first-party trackers.

Reply:
Okay.
Updated at a5212e9d42d2b2144450de1f085e96e42c9951cc.